packages.config

```diff
@@ -1,13 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="DotNetNuke.Core" version="9.4.0" targetFramework="net452" />
-  <package id="DotNetNuke.Instrumentation" version="9.4.0" targetFramework="net452" />
-  <package id="DotNetNuke.Web" version="9.4.0" targetFramework="net452" />
-  <package id="DotNetNuke.Web.Client" version="9.4.0" targetFramework="net452" />
-  <package id="DotNetNuke.Web.Deprecated" version="9.4.0" targetFramework="net452" />
-  <package id="DotNetNuke.WebApi" version="9.4.0" targetFramework="net452" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net452" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net452" />
+  <package id="DotNetNuke.Core" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.DependencyInjection" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Instrumentation" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Web" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Client" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Deprecated" version="9.6.2" targetFramework="net472" />
```

Comment:
I am surprised this was was still there, in a perfect world it would be nice to remove it.